lib/ds_ldap.php

```diff
@@ -204,10 +204,13 @@ protected function connect($method,$debug=false,$new=false) {
 		if (function_exists('run_hook'))
 			run_hook('pre_connect',array('server_id'=>$this->index,'method'=>$method));
 
-		if ($this->getValue('server','port'))
-			$resource = ldap_connect($this->getValue('server','host'),$this->getValue('server','port'));
-		else
-			$resource = ldap_connect($this->getValue('server','host'));
+		$uri = $this->getValue('server','host');
+		if (strpos($uri, '://') === false) {
+			$uri = 'ldap://' . urlencode($uri);
+			if ($this->getValue('server','port'))
+				$uri .= ':' . $this->getValue('server','port');
+		}
+		$resource = ldap_connect($uri);
```

Comment:
I think this solves https://github.com/leenooks/phpLDAPadmin/issues/280

@leenooks do you validate this diff ?
So I can apply it into Debian with more confidence

Reply:
I'd say urls where not supported until now?